app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -6,8 +6,9 @@
   with_border ||= false
   classes = %w(gem-c-layout-footer govuk-footer)
   classes << "gem-c-layout-footer--border" if with_border
+  ga4_index_total ||= nil
```

Comment:
This is for the OGL / Crown Copyright link to know what the `index-total` value is.

Reply:
Is there any scenario where this wouldn't be passed, and it would default to `nil`?

Reply:
@andysellick I think it can be removed, though if the footer ever gets rendered without `ga4_index_total` I think the template would crash without the `||= nil`. Should I remove it?